src/Model/Filter/Like.php

```diff
@@ -147,8 +148,10 @@ protected function _formatWildcards($value)
     protected function _setEscaper()
     {
         if ($this->getConfig('escaper') === null) {
-            /** @var \Cake\Database\Query $query */
```

Comment:
Is the below code still autocompletable in IDE without this? $query->getConnection()->getDriver() etc?